### PR TITLE
Fixes #36094 - handle missing pulp storage-details

### PR DESCRIPTION
**Pull request by m-bucher**

### CODE REVIEW

File: app/models/katello/concerns/smart_proxy_extensions.rb [hunk 1]
```diff
@@ -222,16 +222,29 @@ def pulp_disk_usage
           end
```
[the remark this answers is not in the document]
Comment (m-bucher) [reply app/models/katello/concerns/smart_proxy_extensions.rb]: :+1:

**Comment by sjha4** [comment]
Looks good..the test failure is related. The stubs in the test only stub pulp2 response..

**Comment by m-bucher** [comment]
@sjha4 fixed the test :grin:

**Review by sjha4 — APPROVED** [review]
Ack..Thanks @m-bucher !

**Pull request merged**